charts/adoption-backend/values.yaml

```diff
@@ -3,13 +3,23 @@ tags:
   ccd-idam-pr: false
 
 java:
+  image: hmctspublic.azurecr.io/adoption/backend:latest
   applicationPort: 4550
   ingressHost: adoption-backend-{{ .Values.global.environment }}.service.core-compute-{{ .Values.global.environment }}.internal
   aadIdentityName: adoption
   environment:
-    image: 'hmctspublic.azurecr.io/adoption/backend:latest'
+    IDAM_CLIENT_ID: adoption
+    IDAM_API_URL: https://idam-api.aat.platform.hmcts.net
```

Comment:
I think we can use templated version `{{ .Values.global.environment }}` in place of `aat` as this will return the appropriate environment name